src/BigInt.re

```diff
@@ -1,6 +1,18 @@
 type t;
 
-[@bs.val]
-external fromInt: int => t = "BigInt";
-[@bs.val]
-external toInt: t => int = "Number";
+[@bs.val] external fromInt: int => t = "BigInt";
+[@bs.val] external toInt: t => int = "Number";
+
+external (~-): t => t = "%negfloat";
+external (+): (t, t) => t = "%addfloat";
+external (-): (t, t) => t = "%subfloat";
+external (*): (t, t) => t = "%mulfloat";
+external (/): (t, t) => t = "%divfloat";
+external (%): (t, t) => t = "%modfloat";
+let (**): (t, t) => t = [%raw {|function (a, b) { return (a ** b); }|}];
+external (land): t => t => t = "%andint";
+external (lor): t => t => t = "%orint";
+external (lxor): t => t => t = "%xorint";
+let lnot: t => t = x => x lxor fromInt(-1);
+external (lsl): t => t => t = "%lslint"
+external (asr): t => t => t = "%asrint"
```

Comment:
nice

Reply:
Yeah, I dug up those macros in the Pervasives module :)